aws/resource_aws_sqs_queue.go

```diff
@@ -410,21 +415,17 @@ func resourceAwsSqsQueueRead(d *schema.ResourceData, meta interface{}) error {
 		}
 	}
 
-	tags := make(map[string]string)
-	listTagsOutput, err := sqsconn.ListQueueTags(&sqs.ListQueueTagsInput{
-		QueueUrl: aws.String(d.Id()),
-	})
+	tags, err := keyvaluetags.SqsListTags(sqsconn, d.Id())
+
 	if err != nil {
-		// Non-standard partitions (e.g. US Gov) and some local development
```

Comment:
Can you please keep this comment for the original context of why this is in here? Thanks.